examples/glasplaat.json

```diff
@@ -0,0 +1,171 @@
+{
+  "@context": [
+    "http://iiif.io/api/presentation/3/context.json"
+  ],
+  "id": "https://raw.githubusercontent.com/viaacode/iiif-manifests/IIIF-52-simple-example/examples/glasplaat.json",
+  "type": "Manifest",
+  "behavior": [
+    "individuals"
+  ],
+  "label": {
+    "nl": [
+      "Vlakbeelden Midden Rijk"
+    ]
+  },
+  "metadata": [
+    {
+      "label": {
+        "en": [
+          "Type"
+        ]
+      },
+      "value": {
+        "en": [
+          "Photo"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Date"
+        ]
+      },
+      "value": {
+        "none": [
+          "1900/1960 <dcterms:created^edtf>"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "pid"
+        ]
+      },
+      "value": {
+        "none": [
+          "5d8nc8tp93"
+        ]
+      }
+    },
+    {
+      "label": {
+        "en": [
+          "Identifiers"
+        ]
+      },
+      "value": {
+        "none": [
+          "BIB.GLAS.016209"
+        ]
+      }
+    }
+  ],
+  "thumbnail": [
+    {
+      "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d/full/480,/0/default.jpg",
+      "type": "Image",
+      "format": "image/jpeg",
+      "service": [
+        {
+          "id": "https://images.meemoo.be/iiif/public/OR-0z70w1b/f6/f64922b8a6314067a1a928119060f51659de2ef8245246e3b8676111c9adc34b9bf8c4f666c14a41936dbeb98669fe4d",
+          "protocol": "http://iiif.io/api/image",
+          "profile": "level1"
+        }
```

Comment:
ik vind het `service`-deel hier raar. `protocol` heb ik hier nog nooit zien gebruikt worden. en het is ook raar dat het `level1` is als het het beeld, die dezelfde IIIF Image API service gebruikt, `level2` is.

het wordt hier ook anders gedaan dan op lijn 132 (die een `level2` heeft). die laatste lijkt mij juist.

Reply:
Opdat de client geen level2 calls zou doen blijkbaar.
Die correctie was ik nog vergeten committen, excuus.